locales/en/messages.json

```diff
@@ -202,7 +202,7 @@
     },
 
     "sensorDataFlashNotFound": {
-        "message": "No dataflash <br>chip found",
+        "message": "No dataflash <br />chip found",
```

Comment:
Not sure why we are switching from html syntax to xhtml here. Also, adding unrelated change like this to an already big pull request puts a lot of unnecessary burden onto reviewers and should be avoided by putting it into a separate pull request.

Reply:
Fixed. Will do a sweep in separate PR for double spaces and rename <br /> to <br>